app/assets/stylesheets/application.css

```diff
@@ -104,14 +117,41 @@ a:active {
   margin: 20px 0;
 }
 
-.bio-post-box2 {
-  border: 2px solid black;
-  margin: 2px 3%;
-  padding: 2px 3%;
-}
-
 .prev-next {
   text-align: center;
   margin-bottom: 10px;
   margin-top: auto;
 }
+
+.see-all-button button,
```

Comment:
- [ ] Because you split your `post index` page into pages, you can see that the URL is different from what was is seen in the wireframe but from the screenshot below, you can see that on the post index page, the first post has 6 comments but instead of just 5 comments being shown as seen in the wireframe as a result of using the `recent_comments` method that was created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md)., all 6 comments show. 
- [ ] So kindly fix this and ensure that when a user is on the `posts index page` only 5 recent comments are visible and when on the `posts show page`, all the comments can be visible :wink: 

| Your Design | The Given Wireframe |
|--------|--------|
| ![Screenshot from 2023-08-23 17-34-24](https://github.com/EmiLoKiJu/BlogApp/assets/102699958/6a8578a2-082c-403c-9e23-7fccfc506e56) | ![image](https://github.com/EmiLoKiJu/BlogApp/assets/102699958/e553cfd5-1b9a-424e-affb-5598ac48069b) |